lib/active_storage_validations/content_type_validator.rb

```diff
@@ -31,7 +35,7 @@ def types
       end
     end
 
-    def types_to_human_format
+    def types_to_human_format(types)
```

Comment:
@igorkasyanchuk sorry for the late answer and tackling of this issue!
For the dynamic proc evaluation, we need to check the options for procs on every validation, call them, then continue the validations with the new options. As the options are used in the `types` method to determine the authorized types, we now have to pass the new options to it and the resulting authorized types can be different on each validation - that's why we then pass the types on to the methods which need them.

I renamed the `types` method to `authorized_types` to avoid confusion with variables and parameters. Looks clearer now, I agree. :)